packages/video_player/video_player_avfoundation/darwin/Classes/FVPVideoPlayerPlugin.m

```diff
@@ -71,8 +71,10 @@ - (FVPDisplayLink *)displayLinkWithRegistrar:(id<FlutterPluginRegistrar>)registr
 
 #pragma mark -
 
-@interface FVPVideoPlayer ()
+API_AVAILABLE(macos(10.15))
```

Comment:
@stuartmorgan is this how I should mark the FVPVideoPlayer? Or how else should I fix `AVPictureInPicutreControllerDelegate` to only be available on `macOS`?

Reply:
This is marking everything in the private interface as 10.15+; you'll need to declare this in an extension:

```objc
API_AVAILABLE(macos(10.15))
@interface FVPVideoPlayer (AVPIPDelegate) <AVPictureInPictureControllerDelegate>
@end

@interface FVPVideoPlayer ()
...
```

Reply:
(This will, correctly, cause you to need to adjust other guards later in the code, such as the `pictureInPictureController` property and calls to it.)